### DISCUSSION TITLE
Access the micro sd card installed in the camera over wifi or ethernet

### DISCUSSION THREAD

**Discussion by gigaklatch** [gtxaspec/wz_mini_hacks#591] [Q&A]
Hi there,

Is there any possibility of using this lil' piece of software to gain access to the micro sd card in your camera and download the video files over your LAN connection? Having to physically take out the micro sd card out of the camera every time you need to get the files leaves a great risk of not closing the rubber lid of the micro sd card reader properly and leaving it poorly protected against wetness in all its forms.

Also, are even the video files themselves readily available on the micro sd card to begin with? 

Regards

**Comment by virmaior** [comment]
yes, it's easy to do using SCP (uses the same credential as SSHing into it).

path is going to be `/opt/` to get to the sd card and `/opt/record` is where it stores the files.

I use the following as a cron job to load it hourly: 
https://github.com/virmaior/zakkun-the-wyze/blob/main/za-toru.sh

**Reply by clee604** [reply]
curious to know what kind of speeds you're getting via SCP? I'm getting pretty slow speeds even via ethernet but I'm not sure if it's due to the SCP encryption

**Reply by gigaklatch** [reply]
Thanks a bunch, virmaior! I did manage to fetch an entire folder to my PC over WiFi using SCP in the windows command prompt. Will look more into maybe running a cron job on a Raspberry Pi that will be on the same network as my Wyze cameras.

I myself got slightly over 2 MB/s over WiFi to my WiFi-connected Windows PC. This was with all of the devices connected a crappy old mobile router running on a battery, perhaps using a proper router could have sped it up more?

**Reply by virmaior** [reply]
4.8 to 6.0 MB/s on USB direct
